### DISCUSSION TITLE
#41 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Riely007** [tog-ETH-er/togETHer#17] [General]
Agenda:

Community updates
Fireside discussion

> 会议主题：togETHer
> 会议时间： 21:00-22:00
> 重复周期：每两周 (周一)
> 
> 点击链接入会，或添加至会议列表：
> https://meeting.tencent.com/s/LxNajKWJw4RV
> 
> 会议 ID：891 5098 6775

**Comment by Riely007** [comment]
**以太坊翻译小组**

- 日常 ethereum.org 的翻译

**Comment by bambalados** [comment]
**Crypto Tech Night**

- CTN#15 主题会环绕和大学与区块链。ETA: 6月30日
- 与团队交流CTN后半年的发展

**Comment by 0x68616f79756e** [comment]
ETHPlanet

1.筹备EDCON会议
2.筹备ETAC会议

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

10个PR #772 ～ #782

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## gitcoin grant

- https://gitcoin.co/grants/3414/dapp-learning-developer-group-1

## sharing meeting

成功举办 infura AMA，感谢各个社区的支持

- https://www.youtube.com/watch?v=4yJDUWZskBE&t=789s

近两周举办了2场分享会

- 《Perpetual V1源码解读》by yan 

- 《Perpetual V2原理解析》by 三火
  - https://www.youtube.com/watch?v=TkqA244olJw&t=1609s

## 翻译小组

- 正式发布17集
  - https://mp.weixin.qq.com/s/VfueJprzkwNXKWe1L07RSg

**Comment by winkybupt** [comment]
## 原语里弄
1. 领读活动完成。
2. Primitive Feed https://github.com/PrimitivesLane/PrimitivesFeed

**Comment by bobjiang** [comment]
- GR14 Gitcoin updates
- ENS updates https://snapshot.org/#/ens.eth/proposal/0x056dd964a9fb8da60cfc46d82c3770b09b22212c030c81321df7de9015053082
- weekly newsletter (2-4小时/w)

**Comment by liushooter** [comment]
### Rebase

- 举行了 4 场 rebase 直播分享：3 场关于 dao 治理的light talk，1 场《Web3 项目监管和融资法律》分享，部分内容更新在 [B 站](https://space.bilibili.com/382886213)

- [Web3 日报](https://rebase.network/posts/category/web3daily) 正常更新 14 期

- 更新 [多篇招聘](https://github.com/rebase-network/who-is-hiring)

- 更新多篇 [活动介绍](https://github.com/rebase-network/event-pool)

**Comment by stephanieeee-ho** [comment]
### ECN
1. 完成  decentralized society 的校对发布工作
2. 常规的翻译和周报

**Comment by Emilyxiannv** [comment]
## DAOSquare
举办组织了一个Twitter Space 话题讨论
https://twitter.com/harvei_eth/status/1535954885954265088
